worknenjoy/gitpay
# Script to update order entries status

## Description

- Added the script ```updateAssignsStatus```
- Tasks that were completed before status migration should now be displayed on user profile

## Changes

- ```./scripts.js```

## Issue

Closes #691

## Impacted Area

- Database
- User Profile

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user A
- Create multiple Tasks
- Create / log user B
- Show interest in some of the new tasks
- Log user A
- Choose user B for the tasks
- Log user B
- Accept some of the task invitations
- Using DB manager tool or Terminal manually set the status of the assigns to *null* and save
- Run the Script ```updateAssignsStatus``` with node in the terminal
- Should see a console log that says 'Assigns successfully updated.'
- Double check Assigns table with DB manager tool or terminal to make sure that status was successfully updated.

## Before
### Database before running script: 
![beforeDB](https://user-images.githubusercontent.com/56381169/88251974-6e247b00-cc61-11ea-89bf-355f9e66b30e.png)

### User Profile before running script: 
![beforeProfile](https://user-images.githubusercontent.com/56381169/88251891-1f76e100-cc61-11ea-834f-12e3f27c0726.png)



## After
### Database after running script: 
![afterDB](https://user-images.githubusercontent.com/56381169/88251839-f5252380-cc60-11ea-97fe-bb1262607b65.png)

### User Profile after running script:
![afterProfile](https://user-images.githubusercontent.com/56381169/88251948-56e58d80-cc61-11ea-9f85-0929bcbf241f.png)
